### ISSUE TITLE
The LibriSpeech data set has achieved a word error rate of 2.57%

### ISSUE THREAD

**Issue by yujunlhz** [k2-fsa/k2#843]
Hey Dan:

I read a notes from
https://min.news/en/tech/abc028a742a164bab1675417f2ed68c0.html
 you said that with K2 tool, 
"At present, we have The LibriSpeech data set has achieved a word error rate of 2.57%, and it is still being further optimized."

Is it possible for us to replicate the good result?

Thanks
Mark

**Comment by csukuangfj** [comment]
> Is it possible for us to replicate the good result?


Yes, please refer to our icefall repo: https://github.com/k2-fsa/icefall

See https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/RESULTS.md
